src/makeRobot.js

```diff
@@ -38,7 +38,98 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
```

Comment:
you can remove quotes from keys
also, use simplified syntax
```suggestion
    name,
    wheels,
    version,
```